bridge/client/app/_views/ktb-sequence-view/ktb-sequence-tasks-list/ktb-task-item.component.html

```diff
@@ -12,11 +12,17 @@
     <div fxLayout="row">
       <div fxFlex>
         <div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="5px">
-          <p
-            class="m-0 mt-1 mb-1 mr-2"
-            *ngIf="task.time"
-            [textContent]="task.time | amDateFormat: dateUtil.getTimeFormat()"
-          ></p>
+          <div *ngIf="task.time">
+            <ng-template #overlay>
+              <span>{{ task.time | amDateFormat: dateUtil.getDateTimeFormat() }}</span>
```

Comment:
This one should also be in seconds, to be aligned with the other overlays I guess.